src/js/components/navbars.js

```diff
@@ -85,48 +86,127 @@
 
   var module = angular.module('mobile-angular-ui.components.navbars', []);
 
-  /**
-   * @directive navbarAbsoluteTop
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned top navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-top">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  var global;
```

Comment:
Its a code review, the comment is bound to a line on the diff. So each comment is specific to one line.

I was referring to `var global` specifically.

A name like `var globalNavbarControllerSingleton` would have been better. Long names are not bad, although the usually feel strange. 

Abbreviations and generic names can not be that obvious. Immagine i didn't even know what that line was related to. That may happen to someone editing your code after you.. in order to understand what is `var global` there for he/she should have read and understood all the rest of the code.